cmd/config/internal/commands/cmdcreatesetter_test.go

```diff
@@ -22,8 +22,10 @@ func TestCreateSetterCommand(t *testing.T) {
 		input             string
 		args              []string
 		out               string
+		inputOpenAPI      string
 		expectedOpenAPI   string
 		expectedResources string
+		err               string
```

Comment:
We are following this pattern to have strings as input in table tests to make it easy to add new tests. This implies we are more emphasizing on message and not the error as a whole. For Example: We can assert actualErr.Error().contains(err) rather than strict equals.